### PR TITLE
Double-click to equip/dequip an item

### PR DESCRIPTION
**Pull request by kyleshay**

### CODE REVIEW

**Comment by kyleshay** [comment]
https://www.reddit.com/r/DestinyItemManager/comments/4ulofl/equip_to_current_character/

> What about the ability where you could simply double click on any item to equip it to (or, I'd probably prefer) into the inventory of) your currently equipped character? The same goes for consumables and Materials, etc.

Similar functionality to this, except it would equip/move to your currently 'active' character. Is that better functionality? Thoughts regardless on this PR?

**Comment by SunburnedGoose** [comment]
I like this idea.  It makes sense.  Would be easy to double tap/click items to current character.

**Comment by kyleshay** [comment]
alright, functionality updated to instead move any item that was double clicked to the active character.
if it's equippable it will equip it, if it's equipped it will dequip, if it's a stackable, it will just move it.

also added a info window if someone tries drag/dropping to equip something on their currently active character.

equip

**Comment by bhollis** [comment]
Super cool!

File: app/scripts/store/dimStoreBucket.directive.js [hunk 1]
```diff
@@ -130,6 +131,15 @@
     vm.moveDroppedItem = dimActionQueue.wrap(function(item, equip, $event, hovering) {
       var target = vm.store;
 
+      if (target.id === _.sortBy(dimStoreService.getStores(), 'lastPlayed').reverse()[0].id && equip) {
+        dimInfoService.show('doubleclick', {
```
Comment (bhollis) [inline app/scripts/store/dimStoreBucket.directive.js]: As a note for the future, we may want to extend infoservice to automatically suppress repeated "tip" dialogs even if folks don't click "don't show this again".

File: app/scripts/store/dimStoreItem.directive.js [hunk 2]
```diff
@@ -97,6 +97,15 @@
         });
       }
 
+      vm.doubleClicked = function doubleClick(item, e) {
+        e.stopPropagation();
+        var active = _.sortBy(dimStoreService.getStores(), 'lastPlayed').reverse()[0];
```
Comment (bhollis) [inline app/scripts/store/dimStoreItem.directive.js]: I smell a method on storeservice - `dimStoreService.getActiveStore()`?

Comment (kyleshay) [inline app/scripts/store/dimStoreItem.directive.js]: agreed. we're using that in quite a few places.

**Pull request merged**